pkg/preparer/orchestrate.go

```diff
@@ -165,6 +169,27 @@ func (p *Preparer) handlePods(podChan <-chan pods.PodManifest, quit <-chan struc
 	}
 }
 
+// check if a manifest satisfies the signature requirement of this preparer
+func (p *Preparer) verifySignature(manifest pods.PodManifest, logger logging.Logger) bool {
+	// do not remove the logger argument, it's not the same as p.Logger
+	if p.keyring == nil {
```

Comment:
ie manifest signatures will be ignored.

If the preparer is "authenticated" (ie it has a pgp keyring), then every manifest it receives will have to be signed by one of the keys in that ring, or otherwise the preparer will refuse to deploy it. "unauthenticated" mode is the opposite - signatures will be ignored, unsigned manifests will also be allowed, etc.